include/xtensor/xchunked_array.hpp

```diff
@@ -266,6 +266,7 @@ namespace xt
                                                   const std::string& directory,
                                                   std::size_t pool_size)
     {
+        m_chunks.set_chunk_shape(chunk_shape);
```

Comment:
I don't get the need for this since the call to `resize` below will resize all the chunks.

Reply:
It is the same mistake I was doing, `resize()` will not iterate through the "physical" chunks of the pool, but through the "logical" chunks. The logical chunks are mapped to a smaller number of physical chunks (by default, only one).

Reply:
Ha indeed! Thanks for the explanation!